piikii-application/src/main/kotlin/com/piikii/application/domain/schedule/ScheduleService.kt

```diff
@@ -36,17 +41,6 @@ class ScheduleService(
     ) {
         val schedules = scheduleQueryPort.findSchedulesByRoomUid(roomUid)
         val scheduleIdsToRegister = schedulesToRegister.map { it.id }.toSet()
-        val schedulesToDelete = schedules.filter { it.id !in scheduleIdsToRegister }
-        scheduleCommandPort.deleteSchedules(schedulesToDelete.map { it.id!! })
-    }
-
-    private fun saveSchedules(schedulesToRegister: List<Schedule>) {
-        val schedulesToSave = schedulesToRegister.filter { it.id == null }
-        scheduleCommandPort.saveSchedules(schedulesToSave)
-    }
-
-    private fun updateSchedules(schedulesToRegister: List<Schedule>) {
-        val schedulesToUpdate = schedulesToRegister.filter { it.id != null }
-        scheduleCommandPort.updateSchedules(schedulesToUpdate)
+        scheduleCommandPort.deleteSchedules(schedules.filter { it.id !in scheduleIdsToRegister }.map { it.id!! })
```

Comment:
앗 이거 제말은 변수 뽑은거에서 map까지 뽑는게 어떨까 싶었습니다.
```suggestion
       val scheduleIdToDelete = schedules.filter { it.id !in scheduleIdsToRegister }.map { it.id!! }
        scheduleCommandPort.deleteSchedules(scheduleIdsToDelete)
```

Reply:
아앗 이해이해!! 친절한 설명 감사해요 🥹